pytables.ipynb

```diff
@@ -0,0 +1,417 @@
+    "np.save('data/ctd_adj', ctd_adj)\n",
+    "t2 = time.time()\n",
+    "print(f'{(t2 - t1)*1000} ms')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "5.441904067993164 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Sparse matrices into .npy\n",
+    "t1 = time.time()\n",
+    "np.save('data/sparse_gig_adj', sparse_gig_adj)\n",
+    "np.save('data/sparse_ctd_adj', sparse_ctd_adj)\n",
+    "t2 = time.time()\n",
+    "print(f'{(t2 - t1)*1000} ms')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### sparse.save_npz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "186.43593788146973 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Sparse matrices into .npz with compression\n",
+    "t1 = time.time()\n",
+    "sparse.save_npz('data/sparse_gig_adj', sparse_gig_adj, compressed=True)\n",
+    "sparse.save_npz('data/sparse_ctd_adj', sparse_ctd_adj, compressed=True)\n",
+    "t2 = time.time()\n",
+    "print(f'{(t2 - t1)*1000} ms')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "13.414859771728516 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Sparse matrices into .npz without compression\n",
+    "t1 = time.time()\n",
+    "sparse.save_npz('data/sparse_gig_adj', sparse_gig_adj, compressed=False)\n",
+    "sparse.save_npz('data/sparse_ctd_adj', sparse_ctd_adj, compressed=False)\n",
+    "t2 = time.time()\n",
+    "print(f'{(t2 - t1)*1000} ms')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load matrices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### np.load"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "78.45306396484375 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Dense matrices out of .npy\n",
+    "t1 = time.time()\n",
+    "gig_adj_load = np.load('data/gig_adj.npy')\n",
+    "ctd_adj_load = np.load('data/ctd_adj.npy')\n",
+    "t2 = time.time()\n",
+    "print(f'{(t2 - t1)*1000} ms')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "12.098550796508789 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Sparse matrices out of .npy\n",
+    "t1 = time.time()\n",
+    "sparse_gig_adj_load = np.load('data/sparse_gig_adj.npy')\n",
+    "sparse_ctd_adj_load = np.load('data/sparse_ctd_adj.npy')\n",
```

Comment:
It's confusing for sparse to be in this filename? since it was saved with `numpy.save`?